### PR TITLE
[#131600651] Fix SocialZombie Express Plugin

### PR DESCRIPTION
**Pull request by jonajonlee**
https://www.pivotaltracker.com/story/show/131600651
- SocialZombie Express plugin needs to return the client
- Moved Social Zombie interface to lib/socialzombie

### CODE REVIEW

File: lib/memory/index.js [hunk 1]
```diff
@@ -123,6 +123,9 @@ module.exports = function MemoryStore(data_set) {
 
   });
 
+
+  const getState = R.always(store)
```
Comment (scull7) [inline lib/memory/index.js]: I really don't like this.

Comment (jonajonlee) [reply lib/memory/index.js]: i didn't know it was in there. i'm removing this

Comment (jonajonlee) [reply lib/memory/index.js]: removed

File: lib/rabbitmq/factory.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 
 const R                        = require('ramda')
+const Bluebird                 = require('bluebird')
```
Comment (scull7) [inline lib/rabbitmq/factory.js:3]: I think you need to re-create this PR.

**Pull request closed without merging**